MBBSEmu.Tests/ExportedModules/Majorbbs/strcpy_Tests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using MBBSEmu.Memory;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class strcpy_Tests : MajorbbsTestBase
+    {
+        private const int STRCPY_ORDINAL = 574;
+
+        [Theory]
+        [InlineData("TEST1\0", "TEST1\0")]
+        [InlineData("TEST1\0TEST2\0", "TEST1\0")]
+        [InlineData("\0", "\0")]
```

Comment:
Where's the NULL test? :)

Reply:
3rd test. The tests inits by writing `X` * 0xFF to the destination. Setting the source to null verifies that the destination is now null

Reply:
That's not a NULL pointer though, it's a pointer to an array with \0 as the first element

Reply:
Checked in a test for this and added a little extra handling in `strcpy()` if the source is null